_redirects

```diff
@@ -207,11 +207,14 @@ layout: null
 /docs/v20.2/stop-a-node.html /docs/v20.2/cockroach-quit.html 301
 /docs/v21.1/stop-a-node.html /docs/v21.1/cockroach-quit.html 301
 /docs/stable/stop-a-node.html /docs/stable/cockroach-quit.html 301
-/docs/v21.1/topology-geo-partitioned-replicas.html /docs/v21.1/regional-tables.html 301
+/docs/v21.:version/topology-geo-partitioned-replicas.html /docs/v21.:version/regional-tables.html 301
+/docs/v22.:version/topology-geo-partitioned-replicas.html /docs/v22.:version/regional-tables.html 301
 /docs/stable/topology-geo-partitioned-replicas.html /docs/stable/regional-tables.html 301
-/docs/v21.1/topology-geo-partitioned-leaseholders.html /docs/v21.1/regional-tables.html 301
+/docs/v21.:version/topology-geo-partitioned-leaseholders.html /docs/v21.:version/regional-tables.html 301
+/docs/v22.:version/topology-geo-partitioned-leaseholders.html /docs/v22.:version/regional-tables.html 301
 /docs/stable/topology-geo-partitioned-leaseholders.html /docs/stable/regional-tables.html 301
-/docs/v21.1/topology-duplicate-indexes.html /docs/v21.1/global-tables.html 301
+/docs/v21.:version/topology-duplicate-indexes.html /docs/v21.:version/global-tables.html 301
```

Comment:
ah I guess the way we do redirects changed to use this file
I'm sure you mentioned something about it that I half heard / understood

Reply:
I made a note about this in Docs Weekly on Monday, but it seems like the syntax I used here doesn't work, so I gotta fix it.

Reply:
Redirects fixed.